odl/test/tomo/geometry/geometry_test.py

```diff
@@ -9,102 +9,347 @@
 """Test ODL geometry objects for tomography."""
 
 from __future__ import division
+
+from itertools import permutations, product
 import pytest
 import numpy as np
 
 import odl
-from odl.util.testutils import almost_equal, all_almost_equal, all_equal
+from odl.util.testutils import all_almost_equal, all_equal, simple_fixture
+
+
+# --- pytest fixtures --- #
+
+
+det_pos_init_2d_params = list(set(permutations((1, 0))) |
+                              set(permutations((-1, 0))) |
+                              set(permutations((1, -1))))
+det_pos_init_2d = simple_fixture('det_pos_init', det_pos_init_2d_params)
```

Comment:
Lots of options here, are the tests still running in reasonable time? Overall I've been seeing test runtime growth that is worrying me lately (full suite takes half a minute) so we should be careful here.

Perhaps an exhaustive test should be in largescale?